### ISSUE TITLE
Tabline doesn't work with shadmansaleh's Lualine fork

### ISSUE THREAD

**Issue by Hubro** [kdheepak/tabline.nvim#16]
This is my config:

```lua
  use {
    -- "hoob3rt/lualine.nvim",

    -- Fork that allows reloading config. See: https://github.com/hoob3rt/lualine.nvim/pull/311
    --
    "shadmansaleh/lualine.nvim",

    requires = {
      "nvim-lua/lsp-status.nvim",
      "SmiteshP/nvim-gps",
      "kdheepak/tabline.nvim",
    },
    config = function()
      local lualine = require("lualine")
      local tabline = require("tabline")
      local lsp_status = require("lsp-status")
      local gps = require("nvim-gps")

      tabline.setup { enable = false }

      lsp_status.config{
        diagnostics = false,
        current_function = false,
        status_symbol = "LSP ✓",
      }

      gps.setup{
        icons = {
          ["class-name"] = '⛤ ',      -- Classes and class-like objects
          ["function-name"] = ' ',   -- Functions
          ["method-name"] = ' '      -- Methods (functions inside class-like objects)
        }
      }

      lualine.setup{
        sections = {
          lualine_b = {
            "branch",
            function() return lsp_status.status() end,
          },
          lualine_c = {
            "filename",
            { gps.get_location, condition = gps.is_available }
          },
        },
        tabline = {
          lualine_a = { "filename" },
          lualine_c = { tabline.tabline_buffers },
          lualine_x = { tabline.tabline_tabs },
        }
      }
    end
  }
  -- }}}
```

Notice I use a fork of lualine that allows `config` to be run again to change settings. This fork fixes a whole bunch of issues and will hopefully be merged soon.

(See pull request: https://github.com/hoob3rt/lualine.nvim/pull/311)

However, these two lines:

```
lualine_c = { tabline.tabline_buffers },
lualine_x = { tabline.tabline_tabs },
```

Have absolutely no effect in the forked version of Lualine:

![image](https://user-images.githubusercontent.com/597206/132556191-c9cfccca-1948-468a-a2bb-9b366087b4b7.png)

If I revert to h00b3rt's version, it works fine.

**Comment by kdheepak** [comment]
Thanks for opening the issue. I’m using shadmansaleh’s fork too, although I’m using an explicit tabline. I’ll test this with your config and let you know.

**Comment by kdheepak** [comment]
Should be fixed in https://github.com/kdheepak/tabline.nvim/commit/66aea141aeff769a2d043e7ebf43469aeb9753da.

**Comment by Hubro** [comment]
> This should be fixed in [66aea14](https://github.com/kdheepak/tabline.nvim/commit/66aea141aeff769a2d043e7ebf43469aeb9753da).

Confirmed! Works perfectly :grin: 

Thanks for the quick fix! :rocket:

**Comment by kdheepak** [comment]
Awesome!
